package.json

```diff
@@ -20,7 +20,10 @@
   },
   "license": "MIT",
   "peerDependencies": {
-    "msw": ">=0.35.0"
+    "msw": ">=0.36.9"
```

Comment:
Didn't mean to add this, this is for a separate issue, to upgrade msw to `1.x`

Reply:
It's probably a breaking change, so I think it's ok to do it at the same time as long as it's documented.